packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -274,31 +276,28 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
                         'var promise = new Promise(async function(resolve, reject) {',
                         Template.indent([
                           'installedChunkData = installedChunks[chunkId] = [resolve, reject];',
+                          // filename is duplicated in here and in filesyste strategy below
                           `var filename = require('path').join(__dirname, ${JSON.stringify(
                             rootOutputDir
                           )} + ${
                             RuntimeGlobals.getChunkScriptFilename
                           }(chunkId));`,
+                          'function installChunkCallback(error,chunk){',
+                          Template.indent([
+                            'if(error) return reject(error);',
+                            'installChunk(chunk);',
+                          ]),
+                          '}',
                           'var fs = typeof process !== "undefined" ? require(\'fs\') : false;',
                           'if(fs && fs.existsSync(filename)) {',
                           this._getLogger(
                             `'chunk filename local load', chunkId`
                           ),
                           `loadChunkStrategy('filesystem', chunkId, ${JSON.stringify(
                             rootOutputDir
-                          )}, remotes, function(error,chunk){
-                          if(error) return reject(error);
-                          installChunk(chunk);
-                          })`,
-                          // `DynamicFileSystem(loadChunkFileSystemRunInContext).loadChunk(chunkId,${JSON.stringify(
-                          //   rootOutputDir
-                          // )}, remotes, function(error,chunk){
-                          // if(error) return reject(error);
-                          // installChunk(chunk);
-                          // })`,
+                          )}, remotes, installChunkCallback);`,
                           '} else { ',
                           Template.indent([
-                            loadScriptTemplate,
```

Comment:
No longer depend on any previous script loading code in this repo. All code loading is handled over require.l and don't so internally